lib/redmine.js

```diff
@@ -56,7 +56,7 @@ class Redmine {
    * @param {Callable} callback
    */
   taskEntryFromToggl(data, callback) {
-    let descriptionMatch = (data.description || '').match(/#(\d+)/);
+    let descriptionMatch = (data.description || '').match(/#(\d+)\s?(.+)?/);
```

Comment:
- This would match also `#12345comment`, because `\s?`. Consider using `(\s(.+))?`.
- Toggl integration with redmine creates entries in format `<tracker> #<issue-id> <subject>`, maybe it'd be better to separate subject from comment? With a `-` or `:`?

Reply:
So maybe regex should be like this: #(\d+):?(.+)?

Reply:
Better but still #12345comment will be matched, it is better to use /#(\d+)(\s*:\s*(.+))?/, you'd have to change group index from 2 to 3. The latter regex matches:

- #12345 : comment
- #12345:comment
- #12345 : comment
- etc.

Reply:
Now I see the difference. You are right. I fixed it.